src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""The following are the unpolarized time-like leading order Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from eko import constants
+
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, s1):
+    """
+    Computes the LO quark-quark anomalous dimension
+    Implements Eqn. (B.3) from hep-ph/0604160
+
+    Input parameters
+    ----------------
```

Comment:
These are just called "Parameters":
```suggestion
    Parameters
    ----------
```